en/setup/dev_env_linux.md

```diff
@@ -14,63 +37,113 @@ sudo usermod -a -G dialout $USER
-* Host simulation: [jMAVSim SITL](../simulation/sitl.md) and [Gazebo SITL](../simulation/gazebo.md)
+Ubuntu comes with a serial modem manager which interferes heavily with any robotics related use of a serial port \(or USB serial\). It can removed/deinstalled without side effects:
+
+```sh
+sudo apt-get remove modemmanager
+```
+
+## Ninja Build System
 
-> **Info** Install the [Ninja Build System](../setup/dev_env_linux_boutique.md#ninja-build-system) for faster build times than with Make. It will be automatically selected if installed.
+Ninja is fast than Make and the PX4 CMake generators support it. Unfortunately Ubuntu carries only a very outdated version at this point. To install a recent version of [Ninja](https://github.com/martine/ninja), download the binary and add it to your path:
+
+<div class="host-code"></div>
+
+```sh
+mkdir -p $HOME/ninja
+cd $HOME/ninja
+wget https://github.com/martine/ninja/releases/download/v1.6.0/ninja-linux.zip
+unzip ninja-linux.zip
+rm ninja-linux.zip
+exportline="export PATH=$HOME/ninja:\$PATH"
+if grep -Fxq "$exportline" ~/.profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
+. ~/.profile
+```
+
+## Common Dependencies
+
+Update the package list and install the following dependencies for all PX4 build targets. 
+
+> **Info** Install the [Ninja Build System](#ninja-build-system) for faster build times than with `Make`. It will be automatically selected if installed.
 
 ```sh
 sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
 sudo apt-get update
 sudo apt-get install python-argparse git-core wget zip \
     python-empy qtcreator cmake build-essential genromfs -y
-# simulation tools
-sudo apt-get install ant protobuf-compiler libeigen3-dev libopencv-dev openjdk-8-jdk openjdk-8-jre clang-3.5 lldb-3.5 -y
 # required python packages
 sudo apt-get install python-pip
 sudo -H pip install pandas jinja2
 ```
 
-### NuttX based hardware
+## Simulation Dependencies
+The dependencies for the Gazebo and jMAVSim simulators listed below. You should minimally install jMAVSim to make it easy to test the installation. Additional information about these and other supported simulators is covered in: [Simulation](../simulation/README.md).
 
-Ubuntu comes with a serial modem manager which interferes heavily with any robotics related use of a serial port \(or USB serial\). It can deinstalled without side effects:
+### jMAVSim
 
-```sh
-sudo apt-get remove modemmanager
+Install the dependencies for [jMAVSim Simulation](../simulation/sitl.md).
+
+```
+# jMAVSim simulator
+sudo apt-get install ant openjdk-8-jdk openjdk-8-jre -y
 ```
 
-Update the package list and install the following dependencies. Packages with specified versions should be installed with this particular package version.
+### Gazebo
+
+Install the dependencies for [Gazebo Simulation](../simulation/gazebo.md).
+
+```
+# Gazebo simulator
+sudo apt-get install protobuf-compiler libeigen3-dev libopencv-dev -y
```

Comment:
@LorenzMeier @vilhjalmur89 @ChristophTobler 

> @ChristophTobler @vilhjalmur89 Can we install Gazebo 8 on top / in parallel to ROS?

We can, but just to confirm, do we want to? ROS recommend using gazebo7 in order to be able to "use ROS Kinetic package related to Gazebo from ROS deb repository". If we use gazebo8 we will have to build those from source using catkin workspaces.

1. Can I assume that using gazebo8 and recompiling packages as needed is the best approach for PX4 developers (ie using gazebo7 is not an option and not better if it is)?

----
Assuming we do use gazebo8 the instructions say:

> Install ros-kinetic-gazebo8-ros-pkgs from the osrfoundation repository, which will install the gazebo8 package.

2. Is the following correct (these effectively replace our existing instructions of `sudo apt-get install gazebo8 -y` with `sudo apt-get install ros-kinetic-gazebo8-ros-pkgs -y`) 
```sh
# Gazebo simulator
sudo apt-get install protobuf-compiler libeigen3-dev libopencv-dev -y
sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
## Setup keys
wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
## Update the debian database:
sudo apt-get update -y

## ADD ROS AND GAZEBO *
sudo apt-get install ros-kinetic-gazebo8-ros-pkgs -y
```

3. DO I still need the line below, or is this included?
```
## For developers (who work on top of Gazebo) one extra package
sudo apt-get install libgazebo8-dev
```